terraform/cp4auto/outputs.tf

```diff
@@ -17,6 +17,25 @@ output "kubeconfig" {
   value = data.ibm_container_cluster_config.cluster_config.config_file_path
 }
 
+output "namespace" {
+  value = var.namespace
+}
+
+output "cp4ba_url" {
+  description = "Use this URL to access your Cloud Pak for Business Automation"
+  value       = module.cp4auto.cp4auto_endpoint
```

Comment:
This output file will currently error as the module.cp4auto output from https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/cp4autoTestingDir/cp4auto doesn't exist.

At this point in time, we need to add the output or comment these lines out until it can be resolved